application/src/main/resources/thingsboard.yml

```diff
@@ -991,11 +991,15 @@ transport:
   #        value: "${LWM2M_PROTOCOL_STAGE_THREAD_COUNT:4}"
   snmp:
     enabled: "${SNMP_ENABLED:true}"
+    bind_port: "${SNMP_BIND_PORT:1620}"
```

Comment:
bind port requires changes to the haproxy configuration and load balancers on prod